tests/pytests/test_vecsim.py

```diff
@@ -2394,3 +2394,30 @@ def test_switch_write_mode_multiple_indexes(env):
     if bg_indexing == 0:
         prefix = "::warning title=Bad scenario in test_vecsim:test_switch_write_mode_multiple_indexes::" if GHA else ''
         print(f"{prefix}All vectors were done reindex before switching back to in-place mode")
+
+
+def test_vector_index_ptr_valid(env):
+    conn = getConnectionByEnv(env)
+    # Scenerio1: Vecsim Index scheme with numeric (or non-vector type) and vector type with "bad" parameter 
+    #            Insert partial doc - only numeric
+    #            Update Doc
+    # Note: Should be updated if "bad" parameters are updated
```

Comment:
Let's just explain what is the "bad" parameter scenario that we are testing here